src/Lint.ts

```diff
@@ -1,8 +1,7 @@
-// @flow
```

Comment:
I think it would be a good idea to match the file naming convention in [ast-metadata-inferer](https://github.com/amilajack/ast-metadata-inferer/tree/master/src/helpers).